README.md

```diff
@@ -40,46 +43,34 @@ $ sudo apt update
 $ (. /etc/os-release && sudo dnf install -y https://packages.red-data-tools.org/centos/${VERSION_ID}/red-data-tools-release-latest.noarch.rpm)
 ```
 
-## OpenCV GLib
-
-This section describes how to install
-[OpenCV GLib](https://github.com/red-data-tools/opencv-glib) package.
+## How to install packages
 
-### Debian GNU/Linux and Ubuntu
+### OpenCV GLib
 
 ```console
-$ sudo apt install -y -V libopencv-glib-dev
```

Comment:
Should we remove these options from here?

Reply:
I think these options should be removed.
I think command examples should be given in a simple form. And users should be able to select command line options they want to use. 

However, I admit that this approach is not always the best. 
For example, the installation instructions for [Apache Arrow](https://arrow.apache.org/install/) include the `-y -V` option. These options are useful for copy and paste to execute multiple lines of command at once.

But I don't think they are necessary for short one-line commands such as `apt install fuga`
Some people prefer to check for additional libraries to be installed and then press Y.

Reply:
OK.